pkg/actors/actors.go

```diff
@@ -1216,6 +1219,10 @@ func (a *actorsRuntime) saveActorTypeMetadata(actorType string, actorMetadata *A
 		Key:   metadataKey,
 		Value: actorMetadata,
 		ETag:  actorMetadata.Etag,
+		Options: state.SetStateOption{
```

Comment:
This is a must to make sure first-write behavior is respected when etag is nil (write works only if record does not exist).